File: DemoServer/Indexes/CategoryDescription.cs
using System.Linq;
using DemoServer.Entities;
using Raven.Client.Documents.Indexes;

namespace DemoServer.Indexes
{
    public class CategoryDescription : AbstractIndexCreationTask<Category>
    {
        public CategoryDescription()
        {
            Map = categories => from category in categories
                                select new
                                {
                                    category.Description
                                };

            Stores.Add(x => x.Description, FieldStorage.Yes);
            Analyzers.Add(x => x.Description, "Lucene.Net.Analysis.Standard.StandardAnalyzer");
        }
    }
}

File: DemoServer/JavaDemo/src/main/java/net/ravendb/demo/indexes/CategoryDescription.java
package net.ravendb.demo.indexes;

import net.ravendb.client.documents.indexes.AbstractIndexCreationTask;
import net.ravendb.client.documents.indexes.FieldStorage;

public class CategoryDescription extends AbstractIndexCreationTask {

    public CategoryDescription() {
        map = "from category in docs.Categories " +
                "select new {" +
                "   Description = category.Description" +
                "}";

        store("Description", FieldStorage.YES);
        analyze("Description", "Lucene.Net.Analysis.Standard.StandardAnalyzer");
    }
}
